documentation/frontend/reference/miscellaneous.md

```diff
@@ -86,7 +86,7 @@ If you use VS Code, you can install the
 [Tailwind CSS IntelliSense](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss)
 extension to get autocomplete for TailwindCSS classes.
 
-## Finding your local IP address <a name = "finding-your-local-ip-address"></a>
```

Comment:
This additional tag appears to be leftover from when the documentation was copied over from GitHub READMEs. MyST already parses headers with the exact same naming convention, so it isn't necessary. And whatever it _was_ doing with this, caused the build to break after updating to version 2. This makes sense because the Markdown parser had a major breaking change in its parsing. This is probably a side effect of that. In any case, the fix was easy enough :slightly_smiling_face: